examples/react-app/src/App.tsx

```diff
@@ -42,8 +42,8 @@ function App() {
     <div className="App">
       <h1>Pet List</h1>
       <ul>
-        {Array.isArray(data) &&
-          data?.map((pet, index) => (
+        {Array.isArray(data?.data) &&
```

Comment:
data?.data is ugly right?
Why do we have another object with data inside of it?
This also introduces a breaking change.

Reply:
yup, was planning to dig into this and see this was changed.

Reply:
@seriouslag fixed. not sure how I like the solution but you can you see it in this commit:
https://github.com/7nohe/openapi-react-query-codegen/pull/125/commits/a341432c0cae4196db3c58088b9469c27ed61941